appservice/src/intTest/groovy/nu/yona/server/UserPhotoTest.groovy

```diff
@@ -0,0 +1,290 @@
+/*******************************************************************************
+ * Copyright (c) 2015 Stichting Yona Foundation
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v.2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server
+
+import org.apache.http.HttpEntity
+import org.apache.http.entity.mime.MultipartEntityBuilder
+import org.apache.http.entity.mime.content.InputStreamBody
+import org.codehaus.groovy.runtime.MethodClosure
+
+import groovy.json.*
+import nu.yona.server.test.User
+
+class UserPhotoTest extends AbstractAppServiceIntegrationTest
+{
+	def 'User uploads user photo'()
```

Comment:
Can you name the tests in the familiar pattern with Richard and Bob? This is particularly relevant for tests on buddy interaction.

Reply:
That indeed makes the tests clearer (which roles they play)